core/src/main/java/org/jruby/javasupport/JavaSupport.java

```diff
@@ -147,7 +147,8 @@ public synchronized RubyModule computeValue(Class<?> klass) {
 
     public abstract RubyClass getJavaClassClass();
 
-    public abstract RubyClass getJavaPackageClass() ;
+    public abstract RubyClass getJavaPackageClass();
+    public abstract void setJavaPackageClass(RubyClass javaPackageClass);
```

Comment:
should work if it's package private - not really public API
```suggestion
    abstract void setJavaPackageClass(RubyClass javaPackageClass);
```